mlflow/tracing/display/display_client.py

```diff
@@ -0,0 +1,78 @@
+import json
+from typing import List
+
+from mlflow.entities import Trace
+
+
+def _serialize_trace_list(traces: List[Trace]):
+    return json.dumps(
+        # we can't just call trace.to_json() because this
+        # will cause the trace to be serialized twice (once
+        # by to_json and once by json.dumps)
+        [json.loads(trace.to_json()) for trace in traces]
+    )
+
+
+class IPythonTraceDisplayClient:
```

Comment:
I agree with extracting the display logic from the client, but the term `Client` seems to be a bit overloaded. We have three layers of client (1) MlflowClient (2) TraceClient and now (3) DisplayClient. Can we rename it to sth else like `DisplayHandler`?

Reply:
Yeah I get what you mean, we have a lot of clients haha. I'll make the update!